internal/sync/server.go

```
@@ -0,0 +1,320 @@
+				return
+			}
+
+			if notification.Type == Heartbeat {
+				continue
+			}
+
+			// Notify our listener first
+			s.notifListener.OnNotification(notification)
+
+			// Broadcast to all other clients
+			s.broadcastToOthers(notification, clientID)
```

golangci-lint (1.64.8)

Error return value of `s.broadcastToOthers` is not checked

(errcheck)